src/interface/src/app/shared/form-message-box/form-message-box.component.scss

```diff
@@ -0,0 +1,50 @@
+:host {
+  .message-box {
+    border-radius: 2px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 12px 10px 12px 10px;
+    background-color: rgba(245, 245, 245, 0.4000000059604645);
+    margin-top: 5px;
+    margin-bottom: 1em;
+    font-family: 'Public Sans', 'Roboto', Arial, Helvetica, sans-serif;
+    border: 1px solid black;
+  }
+  .message-box.success {
+    border: 1px solid green;
+  }
+  .message-box.error {
+    border: 1px solid red;
+  }
+
+  :host.message-icon {
+    padding-right: 0.8em;
+    float: left;
+  }
+
+  :host.message-title {
+    text-transform: uppercase;
+    float: left;
+    text-align: left;
+    vertical-align: middle;
+    line-height: 20px;
+    font-size: 1.2em;
+    font-weight: bold;
+    color: black;
+  }
+
+  :host.error .message-title {
+    color: #d50000;
+  }
+
+  :host.message-details {
+    color: #4a4a4a;
```

Comment:
Do you mind doing variables (locally on this file) for the colors, matching the names on figma? That way we can easily swap them once globally present

Reply:
good thought, I can do this on some other branches I'm working on, too